assets/js/search.js

```diff
@@ -2,9 +2,13 @@
 
 window.addEventListener("DOMContentLoaded", () => {
   let searchDialog = document.querySelector(".search-dialog");
+  let searchButton = document.getElementById("search-button");
 
   // Do nothing here if search is not enabled.
-  if (!searchDialog) return;
+  if (!searchDialog || !searchButton) return;
+
+  const isMac = /(Mac|iPhone|iPod|iPad)/i.test(navigator.userAgent);
+  searchButton.title = `Search (${isMac ? "⌘" : "Ctrl"} + K)`;
```

Comment:
This is from https://gist.github.com/DWboutin/7cf15061157f2daf04ef1e25dd110b6a. I don't see any licensing information in there, but that could be omitted since it's a [widely used](https://github.com/search?q=%2F%28Mac%7CiPhone%7CiPod%7CiPad%29%2Fi.test&type=code) snippet.